hoot-services/src/main/java/hoot/services/controllers/grail/GrailResource.java

```diff
@@ -261,79 +261,62 @@ public Response createChangeset(@Context HttpServletRequest request,
         Class<? extends GrailCommand> grailCommandClass;
         GrailParams differentialParams = new GrailParams(reqParams);
         List<Command> workflow = new LinkedList<>();
-        String input1 = reqParams.getInput1();
-        String input2 = reqParams.getInput2();
+        String input1 = reqParams.getInput1() == null ? null : HOOTAPI_DB_URL + "/" + reqParams.getInput1();
+        String input2 = reqParams.getInput2() == null ? null : HOOTAPI_DB_URL + "/" + reqParams.getInput2();
 
-        if (input1 == null && input2 == null) {
-            // Generate command for pulling from the Public Overpass API
-            GrailParams getOverpassParams = new GrailParams(reqParams);
-            InternalCommand getOverpassCommand = getPublicOverpassCommand(jobId, workDir, getOverpassParams);
+        GrailParams getRailsParams = new GrailParams(reqParams);
+        GrailParams getOverpassParams = new GrailParams(reqParams);
```

Comment:
Could these two really be collapsed into a single `grailParams`?
The seem to be used the exact same way below.

Reply:
no because lines 287 & 295 can both be called

Reply:
Or does the `setAdvancedOptions()` on `getRailsParams` make the two not interchangeable.

Reply:
also 305 & 313

Reply:
that's 1 reason but also methods such as `setupRailsPull` and `getPublicOverpassCommand` set an output file

Reply:
Ok, I didn't realize setup was modifying the params.